addons/mail/static/src/core/thread_model.js

```diff
@@ -366,7 +366,19 @@ export class Thread {
                 orderedOnlineMembers.push(member);
             }
         }
-        return orderedOnlineMembers.sort((m1, m2) => (m1.persona.name < m2.persona.name ? -1 : 1));
+        return orderedOnlineMembers.sort((m1, m2) => {
+            const m1HasRtc = Boolean(m1.rtcSession);
+            const m2HasRtc = Boolean(m2.rtcSession);
+            if (m1HasRtc === m2HasRtc) {
+                if (m1HasRtc && m1.rtcSession.isRaisingHand !== m2.rtcSession.isRaisingHand) {
+                    return m2.rtcSession.isRaisingHand - m1.rtcSession.isRaisingHand;
```

Comment:
This implementation is incorrect. Let's say you have 2 users Mitchell Admin and Marc Demo
- Mitchell Admin raises hand => -1 => lowers hand
- Mitchell Admin raises hand (-2) followed by Marc Demo (-1)
=> Mitchell Admin should be before Marc Demo

By using just a persistent integer that just decreases on raising hand, you miss the lowering hand part that should reset timing of following raised hand.

It's safer to use `datetime` of raising hand.

Reply:
@alexkuhn Maybe a misread the scenario, but it looks like it assumes that Marc Demo who raises his hand after, has a higher value in `isRaisingHand`, but the value that is in `isRaisingHand` is obtained from a global variable counter that starts from `Number.MAX_SAFE_INTEGER`.

Admin raising hand => 
`admin.isRaisingHand` = `globalVariable - 1`
Admin raising hand again =>
`admin.isRaisingHand` = `globalVariable - 2`
Marc Demo raising hand =>
`demo.isRaisingHand` = `globalVariable - 3`

Reply:
Updated to a Date & changed the name to `raisingHand`